webapp/channels/src/components/team_controller/team_controller.tsx

```diff
@@ -63,6 +63,10 @@ function TeamController(props: Props) {
     }, []);
 
     useEffect(() => {
+        if (props.disableWakeUpReconnectHandler) {
```

Comment:
`disableWakeUpReconnectHandler` should be a dependency of this hook so that it's started/stopped as the setting is changed